### DISCUSSION TITLE
files app integration

### DISCUSSION THREAD

**Discussion by Fury7425** [moudey/Shell#573] [Ideas]
i feel like people who would care about using nilesoft shell would have interest in useing them together 
https://files.community/

**Comment by JoyHak** [comment]
I also like the idea of integrating the beautiful Nilesoft shell into other file management applications:  Xyplorer file manager, everything search, Flow launcher search, ...

**Reply by boromyr** [reply]
The latest nightly version works in XYplorer.

**Reply by JoyHak** [reply]
> The latest nightly version works in XYplorer.

I can write an `.nss` file that uses `sel.ext` and it will return the file extension?
